1.data-exploration/1.model_data_visualization.ipynb

```diff
@@ -701,10 +701,13 @@
     "# Visualize cancer type distribution\n",
     "cancer_types_bar = (\n",
     "    gg.ggplot(model_df, gg.aes(x=\"OncotreePrimaryDisease\"))\n",
-    "    + gg.geom_bar()\n",
+    "    + gg.geom_bar(width = 0.5, position = position_dodge2(padding = 0.5))\n",
     "    + gg.coord_flip()\n",
+    "    + gg.xlab(\"Primary Disease\")\n",
+    "    + gg.ylab(\"Count\")\n",
     "    + gg.ggtitle(\"Distribution of cancer types\")\n",
     "    + gg.theme_bw()\n",
+    "    + theme(figure_size = (16, 12))\n",
```

Comment:
```suggestion
    "    + gg.theme(figure_size = (16, 12))\n",
```

For consistency.

FWIW, the only reason this works is the imports are a bit wonky:

```python
import plotnine as gg
from plotnine import *  # This line prob should be removed
```